lib/mayaUsd/ufe/UsdShaderAttributeDef.cpp

```diff
@@ -173,7 +173,13 @@ Ufe::Value UsdShaderAttributeDef::getMetadata(const std::string& key) const
 
 #if (UFE_PREVIEW_VERSION_NUM >= 6000)
     if (key == Ufe::AttributeDef::kNativeType) {
-        return Ufe::Value(fShaderAttributeDef->GetType().GetString());
+        // We return the Sdf type as that is more meaningful than the Sdr type.
+        const auto sdfTypeTuple = fShaderAttributeDef->GetTypeAsSdfType();
+        if (!sdfTypeTuple.second.size()) {
```

Comment:
Oh! This is a TfToken, so you want to use `IsEmpty()` instead of the `empty()` I suggested. Using `size()` works, but is frowned upon by clang-tidy as not being as efficient.